assets/locales/en.json

```diff
@@ -61,38 +61,38 @@
+  "感染症の拡大を理由とする従業員の休業等にあたり、国の雇用調整助成金の特例措置等を利用する中小企業等を対象に、社会保険労務士派遣による相談助言を行います（1社5回まで。無料）。": "The Tokyo Metropolitan Government will provide consultation advice by sending a Labor and Social Security Attorney to small and medium-sized enterprise that utilize special measures such as the national employment adjustment subsidy due to COVID-19 (Free of charge up to 5 times per enterprise).",
   "新型コロナウイルス感染症対策雇用環境整備促進事業": "Project for the improvement of the employment environment in response to the COVID-19",
   "感染症の拡大を理由として、国の雇用調整助成金等を活用し、非常時における勤務体制づくりなど職場環境整備に取り組む都内中小企業に奨励金を支給します（1事業所10万円）。": "By utilizing national employment adjustment subsidies due to COVID-19, Tokyo Metropolitan Government will provide finance incentives to small and medium-sized enterprises in Tokyo improving their workplace environment, such as work arrangement in times of emergency.\n(100,000 yen per business site).",
   "テレワーク導入モデル体験事業": "Implementing telework as a trial",
-  "テレワーク導入を検討している中堅・中小企業等を対象に、テレワークを体験できる端末やツールを無償貸与することにより、テレワークのメリット・効果を体感できる機会を提供します。": "For the small and medium-sized enterprises that are considering the implementation of telework, Tokyo Metropolitan Government will lend out terminals and tools free of charge, thus providing the opportunity to experience the merits and effects of telework.",
+  "テレワーク導入を検討している中堅・中小企業等を対象に、テレワークを体験できる端末やツールを無償貸与することにより、テレワークのメリット・効果を体感できる機会を提供します。": "For small and medium-sized enterprises that are considering the implementation of telework, the Tokyo Metropolitan Government will lend out terminals and tools free of charge, thus providing the opportunity to experience the merits and effects of telework.",
   "事業継続緊急対策（テレワーク）助成金": "Subsidies for business continuity measures (telecommuting) ",
-  "都内中堅・中小企業に対し、テレワークの導入に必要な機器やソフトウェア等の経費を助成します（限度額250万円、助成率10/10）。ぜひご活用ください。": "Subsidies (maximum 2.5 million yen, 100% subsidy rate) will be provided to SMEs and other companies in Tokyo for introduction of the equipment, software, etc. necessary for telecommuting. Please make the most of this opportunity.",
+  "都内中堅・中小企業に対し、テレワークの導入に必要な機器やソフトウェア等の経費を助成します（限度額250万円、助成率10/10）。ぜひご活用ください。": "Subsidies (up to a maximum of 2.5 million yen, 100% subsidy rate) will be provided to SMEs and other companies in Tokyo for introduction of the equipment, software, etc. necessary for telecommuting. Please make the most of this opportunity.",
   "テレワークオンラインセミナー開催": "Telework online seminar",
-  "経済団体と連携したテレワーク導入のきっかけづくりのためのオンラインセミナーを開催します。": "In collaboration with Keidanren, Tokyo Metropolitan Government will organize online seminars introducing steps towards telework implementation.",
+  "経済団体と連携したテレワーク導入のきっかけづくりのためのオンラインセミナーを開催します。": "In collaboration with Keidanren, the Tokyo Metropolitan Government will organize online seminars introducing steps towards telework implementation.",
```

Comment:
These will be changed soon from issue #116

Reply:
@amouro okay, I will update accordingly after the changes are reflected. Thank you.